docs/mast/mast_obsquery.rst

```diff
@@ -284,7 +284,7 @@ with `~astroquery.mast.ObservationsClass.get_product_list`
 (see `here <https://mast.stsci.edu/api/v0/_c_a_o_mfields.html>`__ for more details).
 Using "obs_id" instead of "obsid" from the previous example will result in the following error:
 
-.. doctest-remote-data::
+.. doctest-skip::
```

Comment:
why switch to skip for the whole example here?

Reply:
Latest commit doesn't skip the test, but ignores the output where the `RemoteServiceError` occurs.